test/testUtils.ts

```diff
@@ -178,6 +178,37 @@ function triggerFakeMouseEvent(type: string, target: D3.Selection, relativeX: nu
   target.node().dispatchEvent(e);
 }
 
+function triggerFakeTouchEvent(type: string, target: D3.Selection, relativeX: number, relativeY: number) {
```

Comment:
Just mirroring `triggerFakeMouseEvent()`. You're right that it doesn't have to be a `D3.Selection` though.